tests/conftest.py

```diff
@@ -0,0 +1,9 @@
+import pytest
+
+from modules.heatpump import HeatPump
+
+
+@pytest.fixture(scope="function")
+def hp_5kw_24h() -> HeatPump:
+    """Heat pump with 5 kw heating output and 24 h prediction"""
+    return HeatPump(5000, 24)
```

Comment:
why not put it in `test_heatpump` to keep the related things together?

Reply:
Thought we can place all fixture in one place. Maybe adding a subfolder `fixtures` inside `tests` which can be used for that.